portfolio/src/main/webapp/themes/index.js

```diff
@@ -0,0 +1,13 @@
+const WoodSmoke = '#0B0C10';

```

Comment:
I believe JS use all CAP for constants: https://www.google.com/search?q=js+const+naming+convention

Reply:
I didn't know that there was a JS code convention. Thanks, I'll check it out !